### PR TITLE
Post commit Hooks to set the revision

### PR DESCRIPTION
**Pull request by RedBeardCode**
Hello,
I tried a lot but couldn't convince docker resolve the git rev-parse command during the build. So I used the git post-commit hook. If some checks out the repro created with painless he has to set a symbolic link or for newer git versions (>2.9) set the hooksPath parameter.

Best regards,
RedBeardCode

### CODE REVIEW

**Comment by bittner** [comment]
First of all, thanks for placing the very first PR! Thank you.

I've thought about the commit-hook solution, and I feel there must be a better way. The CI server knows which revision (SHA hash) of the repository it checked out. We should use that information right in the docker image build later (as soon as we have this readily working).

We should keep things simple and keep complexity out of the development environment.

I know this feels unfair to ask, but can you remove the changes to the `post_gen_project` hook and the hook script too? Let's keep just the simplification of the settings, and come back to this idea of setting the `REVISION` environment value later in time. Sorry!

**Comment by RedBeardCode** [comment]
No worries. I delete the hooks. Hope it works.

**Comment by bittner** [comment]
Thanks! One thing left: `flake8` complains
```
./hooks/post_gen_project.py:122:1: W391 blank line at end of file
```
Can you fix that, too?

**Comment by RedBeardCode** [comment]
Done.

**Comment by bittner** [comment]
Almost! Now the tests are failing for [the line ensuring the git-related settings for monitoring](https://github.com/painless-software/painless-continuous-delivery/blob/master/tests/unit/test_monitoring.py#L20).

Can you change that, so it matches [the new implementation](https://github.com/painless-software/painless-continuous-delivery/pull/12/files#diff-c2ff4f6aba93d62ed9a32a6dda454da6R131)? (... and when you're at it, can you also change [the comment on line 32](https://github.com/painless-software/painless-continuous-delivery/blob/master/tests/unit/test_monitoring.py#L32)?)

Thanks so much!

**Comment by RedBeardCode** [comment]
So. Now I hope the grass looks green.

File: {{cookiecutter.project_slug}}/_/frameworks/Django/application/settings.py [hunk 1]
```diff
@@ -2,11 +2,7 @@
 Django settings for application project.
 """
 from os.path import abspath, dirname, join
-{%- if cookiecutter.monitoring == 'Sentry' %}, pardir{% endif %}
 from environ import Env
-{% if cookiecutter.monitoring == 'Sentry' %}
-import raven
-{% endif %}
```
Comment (bittner) [inline {{cookiecutter.project_slug}}/_/frameworks/Django/application/settings.py]: Well spotted, thank you!

**Comment by bittner** [comment]
Thanks again!

We now need to come back and add setting `REVISION` on the built images. When the time is due.

**Pull request merged**